### ISSUE TITLE
Pitching a feature

### ISSUE THREAD

**Issue by Shailevy** [yayaa/Rotatable#1]
Great library but I think you missing the real "killer feature" it can provide.
If you add support for automatic rotation according to the device sensors it will allow developer to liven up their apps.
While I know it can't get to be as good as seene ( https://play.google.com/store/apps/details?id=com.obviousengine.seene.android.core&hl=en ) without several images, I still think they nailed a great concept and will be interesting to try and do something similar with rotations and skewing of a single image

**Comment by yayaa** [comment]
Yes, this is actually nice idea. I thought this library works as user's manual interactions as touch but auto-rotate up to gyroscope might be cool. I'll consider this.
